### DISCUSSION TITLE
Impossibility to install dependancies

### DISCUSSION THREAD

**Discussion by NellyDubois** [gatsbyjs/gatsby#39032] [Help]
I create a new Gatsby project (with JS, tailwind). I would like to install other dependancies like gatsby-plugin-react-helmet for example.
Unfortunately I have error messages : 
![image](https://github.com/gatsbyjs/gatsby/assets/143407909/407fd3be-1a05-4aef-8938-38f32d0333a6)
I've tried 
```
npm audit fix
```
```
npm audit fix --force
```

```
npm update
```
but unfortunately it doesn't solve the problems.
I've tried to delete package.lock.json and node modules before reinstalling nm but it doesn't change anything...I have alos tried 
```npm install --legacy-peer-deps```

My version of Gatsby is 5.13.5, Node 20.15.0, npm 10.7.0

How could I solve these issues given that I have other dependancies to install bacuse I woulmd like to maigrate my React website to Gatsby.

Thanks in advance for your help

**Comment by 808vita** [comment]
Gatsby comes 5 comes with `head api` , may be migrate (if it is a viable option).
https://www.gatsbyjs.com/docs/reference/built-in-components/gatsby-head/
